components/FeedbackForm.js

```diff
@@ -17,13 +17,13 @@ export default function FeedbackForm() {
             </label>
         </p>
   
-        <label for="name">Name</label>
+        <label htmlFor="name">Name</label>
         <input className={styles['form-field']} type="text" name="name" />
 
-        <label for="name">Email</label>
+        <label htmlFor="name">Email</label>
         <input className={styles['form-field']} type="email" name="email" required />
 
-        <label for="feedback">What is your feedback?</label>
+        <label htmlFor="feedback">What is your feedback?</label>
```

Comment:
I believe that `htmlFor`/`for` need to be associated by an `id` for the input. If we can add the respective id attributes to each input this should be good!

Reply:
Thanks! I added id's, does that look right?